CLUE_Hand_Wash_Timer/clue_hand_wash_timer.py

```diff
@@ -0,0 +1,88 @@
+"""
+Start a 20 second hand washing timer via proximity sensor.
+Countdown the seconds with text and beeps.
+Display a bitmaps for waiting and washing modes.
+"""
+
+import time
+import board
```

Comment:
You could avoid importing board and just use `clue.display`

Reply:
that changes the display calls throughout right?

Reply:
Oh, nevermind, I see what you mean. Got it, will do.

Reply:
hmm, seems board needs to be imported for buzzer to work, unless there's a clue equivalent to board.SPEAKER

Reply:
I'll switch to `clue.display` since it is more consistent with the other display stuff here, but leaving board import in because it's needed for the way buzzer (pulseio) is being used without a duration. Seems` clue.playTone()` would require a duration and possibly add an unwanted delay to the countdown.

Reply:
Oh yeah, good call. Go ahead and leave it in.